[DOC] Added link to official docs (#109)
neurobagel/proc_dash: README.md

> Quickstart: https://dash.neurobagel.org/

Reviewer: ```suggestion
**Quickstart**: https://dash.neurobagel.org/
```

sneaking this in

> Please also refer to our official [**neurobagel documentation**](https://neurobagel.org/nipoppy/configs/) to learn more about how `proc_dash` can be used together with the nipoppy project.

Reviewer: ```suggestion
For more information on how `proc_dash` can be used together with the Nipoppy project, also see the official [Nipoppy documentation](https://neurobagel.org/nipoppy/overview/).
```

I think we could also move this above the Quickstart. 🙂
